app/assets/stylesheets/custom.css.scss

```diff
@@ -56,17 +56,34 @@
 }
 
 // Organization details page
-// Google maps
 
 img {
   max-width: none;
+  margin: 2em;
 }
 
-td.got-it {
-  border-left: 2px solid black;
+.got-it {
+  border-left: 1px solid #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.